tests_common/setup.py

```diff
@@ -22,7 +22,7 @@
 requires = [
     "asyncpg",
     "click",
-    "inmanta-core~=4.0.dev",
+    "inmanta-core==4.1.0",
```

Comment:
should we factor this out into a variable? 

```
version = "4.1.0"

requires = [
  f"inmanta-core=={version}",
]

setup(
    version=version,

Reply:
Sounds like a good idea.